### PR TITLE
[TEVA-1517] add jobseeker account job alert table markup

### PR DESCRIPTION
**Pull request by alexbowen**
Screenshot 2020-12-02 at 15 53 57

### CODE REVIEW

File: app/views/jobseekers/subscriptions/index.html.haml [hunk 1]
```diff
@@ -3,8 +3,31 @@
...
+                = 'value'
+              %li
+                %span{ class: 'govuk-!-font-weight-bold' }= 'key'
+                = 'value'
+
+          %td.govuk-table__cell= 'Daily, at approx 3pm'
+          %td.govuk-table__cell.govuk-table__header--numeric
+            %ul.govuk-list
+              %li
+                %a.govuk-link{ href: '/' }= 'Manage alert'
```
Comment (cpjmcquillan) [inline app/views/jobseekers/subscriptions/index.html.haml:29]: We tend to use `link_to` helpers (and soon `govuk_link_to` helpers)

Comment (alexbowen) [reply app/views/jobseekers/subscriptions/index.html.haml:29]: ok i see that as being added a subsequent PR though

File: config/locales/jobseekers.yml [hunk 2]
```diff
@@ -43,6 +43,11 @@ en:
         page_title: Saved jobs
     subscriptions:
       index:
+        heading_actions: Actions
```
Comment (cpjmcquillan) [inline config/locales/jobseekers.yml:46]: If I was being super anal I might say we could nest these like
```
heading:
  actions: Actions
``` 
etc, but in reality I don't mind too much

**Review by cpjmcquillan — APPROVED** [review]

File: app/views/jobseekers/subscriptions/index.html.haml [hunk 3]
```diff
@@ -3,8 +3,29 @@
 = render partial: 'account_header'
 
 .govuk-width-container
-  .govuk-grid-row
+  .govuk-grid-row{ class: 'govuk-!-margin-left-0 govuk-!-margin-right-0' }
     .govuk-grid-column-full-width
```
Comment (cpjmcquillan) [inline app/views/jobseekers/subscriptions/index.html.haml:7]: I think this class is actually called `govuk-grid-column-full`

File: app/views/jobseekers/subscriptions/index.html.haml [hunk 4]
```diff
@@ -3,8 +3,29 @@
 = render partial: 'account_header'
 
 .govuk-width-container
-  .govuk-grid-row
+  .govuk-grid-row{ class: 'govuk-!-margin-left-0 govuk-!-margin-right-0' }
```
Comment (cpjmcquillan) [inline app/views/jobseekers/subscriptions/index.html.haml:6]: Which would then mean you don't need these margin classes (I think)

**Pull request merged**